dataactbroker/handlers/accountHandler.py

```diff
@@ -606,9 +606,10 @@ def list_users(self):
                 filter(CGAC.cgac_code == user.cgac_code).\
                 one_or_none()
 
+            permission = None if user.permission_type_id is None else PERMISSION_TYPE_DICT_ID[user.permission_type_id]
```

Comment:
Shouldn't hold this back, but this is a little more concise (it'll set permission to `None` if it can't be found in the dict)
```python
permission = PERMISSION_TYPE_DICT_ID.get(user.permission_type_id)
```